test/azul_test_case.py

```diff
@@ -1,9 +1,50 @@
-import os
-from unittest import TestCase, mock
+from unittest import TestCase
 
 import boto3.session
 
 
+class AlwaysTearDownTestCase(TestCase):
+    """
+    SafeTestCase makes sure that tearDown / cleanup methods are always run when
```

Comment:
```suggestion
    AlwaysTearDownTestCase makes sure that tearDown / cleanup methods are always run when
```

That's why I avoid duplicating class and method names in doc strings. So really,

```suggestion
    Makes sure that tearDown / cleanup methods are always run when
```

Reply:
Missed this one.